src/vsr/superblock.zig

```diff
@@ -1325,6 +1332,7 @@ pub fn SuperBlockType(comptime Storage: type) type {
                 if (superblock.working.manifest_size > 0) {
                     assert(superblock.manifest.count > 0);
                 }
+                // TODO Make the FreeSet encoding format not dependant on the word size.
```

Comment:
Ah nice, are you thinking this would then clean up our FreeSet encoding here in SuperBlock to be the same as the Manifest?

Reply:
How do you plan to do this for FreeSet's encoding format, not to be dependent on word size?

Reply:
> Ah nice, are you thinking this would then clean up our FreeSet encoding here in SuperBlock to be the same as the Manifest?

Actually I'm not sure what you mean by this :thinking:.

> How do you plan to do this for FreeSet's encoding format, not to be dependent on word size?

I don't have a specific plan yet :grin: — it will probably just involve casting the bitset's data to `[]u64` before encoding/decoding, but it has been a while since I worked on that code.
I added this note because we will need it for 32-bit compatibility (a requirement for the WASM work).

Reply:
Ah, I meant that, at present, an empty free set trailer still has a machine word in it, whereas, an empty manifest trailer is completely empty. So the encoding/decoding routines in the superblock are a little different IIRC.

Reply:
Ah, I didn't realize that!